From: Shengjing Zhu <zhsj@debian.org>
Date: Sun, 24 Oct 2021 23:46:43 +0800
Subject: Disable TestWebList

Bug: https://github.com/google/pprof/issues/650
---
 internal/report/source_test.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/internal/report/source_test.go b/internal/report/source_test.go
index 1524eaf..d49f1ab 100644
--- a/internal/report/source_test.go
+++ b/internal/report/source_test.go
@@ -30,6 +30,7 @@ import (
 )
 
 func TestWebList(t *testing.T) {
+	t.Skip("https://github.com/google/pprof/issues/650")
 	if runtime.GOOS != "linux" || runtime.GOARCH != "amd64" {
 		t.Skip("weblist only tested on x86-64 linux")
 	}
